manifests/params.pp

```diff
@@ -53,6 +53,7 @@
       if $::operatingsystem == 'Ubuntu' {
         $libruby_compare = '11'
         $ruby18_compare = '11'
+        $ruby23_compare = '16'
       } else {
         $libruby_compare = '6'
         $ruby18_compare = '6'
```

Comment:
I think for Debian we should define `ruby23_compare` as '9'

https://www.debian.org/releases/
https://packages.debian.org/search?keywords=ruby2.3